students/mariusz_michalczyk/lesson_01_basics/Apple_sharing.py

```diff
@@ -0,0 +1,8 @@
+n = 0
+while (n < 1):
+    n = int(input("Enter amount of students: "))
```

Comment:
I wouldn't care, you can still crash with `ValueError` by supplying string. In the following lessons we will be investigating exceptions.